### ISSUE TITLE
Keeping the book clean

### ISSUE THREAD

**Issue by FelixErnst** [microbiome/OMA#52]
Hi 

I noticed a lot of pecuilar things, which will be very difficult to fix later on.

1. There are two `Rmd` files which begin with `14`, `14a` and `14b`. This is not good and does not follow the setup of the book.
2. 
![image](https://user-images.githubusercontent.com/17824332/126875710-f575aafd-0dae-46b4-b8a4-9d8c69f661b5.png)

I don't know, where the second session info comes from in the middle, but it not helpful nor necessary.

3. There seems to be a plotting section after the bibliography. I think that doesn't make sense
![image](https://user-images.githubusercontent.com/17824332/126875787-fd0eded1-6611-4b1a-a7d4-8871055f14d2.png)

I think these needs to be cleaned up, before the book becomes unmaintainable and unreadable.

**Comment by antagomir** [comment]
Yes. @ChouaibB can you make a PR to fix the obvious errors overlined above (sections in wrong places)

The 14a/14b can be changed into 14 & 15; and other numbers can be bumped up correspondingly.. we will have to work out the numbering scheme in more detail but this is to fix the immediate issues.

You can run in R bookdown::serve_book()

**Comment by antagomir** [comment]
Hmm the "Interactive 3D plots" is still hanging around after Chapter 13. @ChouaibB can you remove that from the side panel?

**Comment by ChouaibB** [comment]
I gave it a try to have it as a hidden page that can be rendered with the automatic flow of github so as it is available as html page when referred to at [the section](https://microbiome.github.io/OMA/beta-diversity.html#visualizing-the-most-dominant-genus-on-pcoa), but unfortunately I did not figure it out yet.
@TuomasBorman suggested to have an Extras chapter [at the discussion](https://github.com/microbiome/OMA/pull/54).
I can try solving it still.
